templates/ai-ml.html

```diff
@@ -1,374 +1,417 @@
-      <h3 class="p-card__title">ベアメタルAI</h3>
-      <hr>
-      <p>NVIDIA GPGPUアクセラレーションによるベアメタルでのKubernetes</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">最高レベルのパフォーマンス</li>
-        <li class="p-list__item is-ticked">ローカルデータを使用したオンプレミス</li>
-        <li class="p-list__item is-ticked">ハードウェアの推奨</li>
-        <li class="p-list__item is-ticked">フルマネージドオプション</li>
-      </ul>
-    </div>
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-6 p-card">
-      <h3 class="p-card__title">Google Cloud AI</h3>
-      <hr>
-      <p>NVIDIA GPGPUアクセラレーションによるUbuntuでのGKE</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">事実上無限の拡張性</li>
-        <li class="p-list__item is-ticked">移行可能なワークロード</li>
-        <li class="p-list__item is-ticked">最速レベルのクラウド機械学習</li>
-      </ul>
     </div>
-    <div class="col-6 p-card">
-      <h3 class="p-card__title">Canonical Cloud AI</h3>
-      <hr>
-      <p>NVIDIA GPGPUアクセラレーションによるOpenstackでのKubernetes上のKubeflow</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">OpenStackが持つ利点の最大化</li>
-        <li class="p-list__item is-ticked">ローカルデータを使用したオンプレミス</li>
-        <li class="p-list__item is-ticked">ハードウェアの推奨</li>
-        <li class="p-list__item is-ticked">フルマネージドオプション</li>
-      </ul>
+  </section>
+
+  <section class="p-section">
+    <div class="row p-section--shallow">
+      <hr />
+      <h2>オープンソースのMLOpsツール</h2>
     </div>
-  </div>
-</section>
-<section class="p-strip--light is-bordered">
-  <div class="row u-equal-height">
-    <div class="col-4 u-vertically-center u-align--center u-hide--small u-hide--medium">
-      <img src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/da5a7c9e-Kubeflow-Logo.svg" alt="">
+    <div class="p-equal-height-row p-section--shallow">
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Charmed Kubeflow</h3>
+        </div>
+        <p class="p-equal-height-row__item">
+          自動化されたツールでモデルを開発およびデプロイします。Charmed Kubeflowは大規模なAI運用を目的としたエンドツーエンドのMLOpsプラットフォームです。Canonical MLOpsの基盤として、他のビッグデータツールや機械学習ツールをシームレスに統合します。
+        </p>
+        <div class="p-equal-height-row__item">
+          <hr class="is-muted" />
+          <p>
+            <a href="https://ubuntu.com/ai/what-is-kubeflow">Kubeflowとは</a>
+          </p>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Charmed MLflow</h3>
+        </div>
+        <p class="p-equal-height-row__item">
+          実験を追跡し、モデルカタログ全体を把握しましょう。Charmed MLFlowは、機械学習のワークロードを管理するオープンソースのプラットフォームであり、他のMLOpsツールと統合して機械学習ライフサイクルのさまざまな機能に対応します。
+        </p>
+        <div class="p-equal-height-row__item">
+          <hr class="is-muted" />
+          <p>
+            <a href="https://ubuntu.com/blog/charmed-mlflow-beta">MLFlowの詳細</a>
```

Comment:
Feel free to confirm with Sophie, but I believe these are all supposed to have the `>` next to them
```suggestion
            <a href="https://ubuntu.com/blog/charmed-mlflow-beta">MLFlowの詳細&nbsp;&rsaquo;</a>
```